src/__experimental__/components/checkbox/checkbox-group.component.js

```diff
@@ -2,49 +2,50 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import tagComponent from '../../../utils/helpers/tags';
 import { StyledCheckboxGroup } from './checkbox.style';
-import { withValidation } from '../../../components/validations';
 import FormField from '../form-field';
 
 const CheckboxGroup = (props) => {
   const {
     children,
-    groupName,
-    hasError,
-    hasWarning,
-    hasInfo
+    id,
+    value,
+    defaultValue
   } = props;
-
-  const groupLabelId = `${groupName}-label`;
+  const defaultCheckedValue = (defaultValue !== undefined) ? defaultValue : [];
+  const checkboxGroupValue = (value !== undefined) ? value : defaultCheckedValue;
+  const groupLabelId = `${id}-label`;
 
   const buttons = React.Children.map(children, (child) => {
+    let checked = false;
+    let defaultChecked = false;
+
+    if (checkboxGroupValue.length > 0) {
```

Comment:
We don't want to support a controlled checkbox-group at this time. We want users to control each checkbox instead. Please can you remove the code relating to`defaultChecked` and `value`

Reply:
I think we can remove the `React.Children.map` entirely.

Reply:
Removed in 2c1a6c0 :thumbsup:

> We don't want to support a controlled checkbox-group at this time.

So does [`fixtures/src/components/controlled/checkbox-group.js`](https://github.com/Sage/carbon/blob/FE-2248-checkbox-group-interface/fixtures/src/components/controlled/checkbox-group.js) need to be deleted then?

Reply:
We can either keep it to show how to use a controlled checkbox in a checkbox-group.
OR
We can update `fixtures/src/components/controlled/checkbox.js ` to show with and without a group.

Reply:
Of those two options, I think this one seems better:

> We can update `fixtures/src/components/controlled/checkbox.js ` to show with and without a group.

I'll do that, and I'll also delete `fixtures/src/components/controlled/checkbox-group.js`.

Reply:
`fixtures/src/components/controlled/checkbox-group.js` removed in 99ebdf3 and ca02d94

CheckboxGroup example added to `fixtures/src/components/controlled/checkbox.js` in 5aee4e4 :thumbsup: